README.md

```diff
@@ -82,17 +141,16 @@ For example:
 ```sh
 #!/bin/sh
 
+cask "dropbox"
```

Comment:
I think we don't use dropbox for file sharing

Reply:
Yes, this line is an example for customizations.